client/my-sites/stats/stats-subscribers-page/subsriber-highlights.jsx

```diff
@@ -0,0 +1,97 @@
+import { HighlightCard, Popover } from '@automattic/components';
+import { Icon, info } from '@wordpress/icons';
+import { translate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import '../wordads/highlights-section.scss';
+
+function getSubscriberHighlights() {
+	const highlights = [
+		{
+			id: 1,
+			heading: 'Total email subscribers',
+			count: 1410,
+			previousCount: 1080,
+		},
+		{
+			id: 2,
+			heading: 'Free email subscribers',
+			count: 1340,
+			previousCount: 1034,
+		},
+		{
+			id: 3,
+			heading: 'Paid email subscribers',
+			count: 70,
+			previousCount: 46,
+		},
+		{
+			id: 4,
+			heading: 'WordPress.com subscribers',
+			count: 322,
+			previousCount: 326,
+		},
+	];
+	return highlights;
+}
+
+function SubscriberHighlightsHeader() {
+	const [ isTooltipVisible, setTooltipVisible ] = useState( false );
+	const infoReferenceElement = useRef( null );
+	const localizedTitle = translate( 'All time stats', {
+		comment: 'Heading for Subscribers page highlights section',
+	} );
+	const showNotices = true;
```

Comment:
`showNotices` should either be a property, or the tooltip should always be visible. Since the designs contain a tooltip, let's always show it and remove the variable.

Reply:
Oh my! I meant to remove this. I was playing with refactoring the heading bits and left this in. I'll replace it with a TODO comment for now.